CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 1.16.0 (September 3, 2024)
+
+ * [#4151](https://github.com/redhat-developer/vscode-openshift-tools/issues/4151) Create a deployment using a BuildConfig
+ * [#4108](https://github.com/redhat-developer/vscode-openshift-tools/issues/4108) Introduced the devfile for .NET 9
```

Comment:
```suggestion
 * [#4108](https://github.com/redhat-developer/vscode-openshift-tools/issues/4108) Introduced support for the devfile for .NET 9.

```

Reply:
Fixed.

Note: Normally we no ending dots at the end of the list items.  If you want it, I can add the "ending dots"